backend/tests/integration/records/serializers/json/test_literature.py

```diff
@@ -1214,9 +1214,14 @@ def test_literature_raw_json_with_logged_in_cataloger(inspire_app):
         response = client.get(f"/literature/{record_control_number}", headers=headers)
 
     response_status_code = response.status_code
-
+    result_keys = response.json().keys()
```

Comment:
json here is not a method but a property.

Reply:
You are right, I mixed up `requests` with flask requests :)